scripts/g_r.py

```diff
@@ -0,0 +1,60 @@
+import datetime
```

Comment:
This still doesn't solve the original issue, It needs to be more descriptive. I should be able to read your code from having previous interactions with it and understand the general idea of what this class/function is for. `gr` or `g_r` does not accomplish this